great_expectations/data_context/store/in_memory_store_backend.py

```diff
@@ -0,0 +1,83 @@
+from typing import Optional
+
+from great_expectations.core.data_context_key import DataContextVariableKey
+from great_expectations.data_context.data_context_variables import (
+    DataContextVariableSchema,
+)
+from great_expectations.data_context.store.store_backend import StoreBackend
+from great_expectations.exceptions import InvalidKeyError
+from great_expectations.util import filter_properties_dict
+
+
+class InMemoryStoreBackend(StoreBackend):
+    """Uses an in-memory dictionary as a store backend."""
+
+    # noinspection PyUnusedLocal
+    def __init__(
+        self,
+        runtime_environment=None,
+        fixed_length_key=False,
+        suppress_store_backend_id=False,
+        manually_initialize_store_backend_id: str = "",
+        store_name=None,
+    ) -> None:
+        super().__init__(
+            fixed_length_key=fixed_length_key,
+            suppress_store_backend_id=suppress_store_backend_id,
+            manually_initialize_store_backend_id=manually_initialize_store_backend_id,
+            store_name=store_name,
+        )
+        self._store = {}
+        # Initialize with store_backend_id if not part of an HTMLSiteStore
```

Comment:
I'm not sure what this comment is trying to say. We isn't an HTMLSiteStore and there is no checking around this. Can we delete it?